### DISCUSSION TITLE
Chrome 129.0.6668.59 (Windows) Does Not Execute remoteEntry.js file from remote app when running host app in dev mode

### DISCUSSION THREAD

**Discussion by makara4code** [originjs/vite-plugin-federation#628] [Q&A] [answered]
When serving host app in dev mode it shows white screen in chrome but work normal in Microsoft Edge.

**Comment by gbcerna97** [comment]
Hi @makara4code, as a temporary solution, I am running my dev server on Firefox since it doesn't work on both Edge and Chrome. When serving the app build, it works just fine on any browser. So, this might be browser related issue as mentioned here #630.

**Comment by makara4code** [comment, marked as answer]
With latest release of chrome v130 this issue is resolved
